### ISSUE TITLE
Storage disks unreadable following 1.15.2 -> 1.16.4 server upgrade

### ISSUE THREAD

**Issue by pwlgrzs** [refinedmods/refinedstorage#2829]
#### Issue description:
Storage/Fluid disks do not contain items following server MC version bump.
#### What you expected to happen:
Storage disks should contain preupgrade items

#### Steps to reproduce:
1. Upgrade MC from 1.15.2 to 1.16.4
2. Copy refinedstorage_disks.dat file to new instance
3. Disks are empty in game and Disk Drives do not allow to insert them
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.16.4
- Forge: 35.1.36
- Refined Storage: 1.9.11

Does this issue occur on a server? 
Yes

We got a few items saved in personal directory, but few players would be super happy if we could recover their drives. Is there any chance to have it working or items are lost in the new version? I will be grateful for any assistance. Thanks!

**Comment by embeddedt** [comment]
Looks like a duplicate of #2752.

TL;DR Modded servers are not designed to be upgraded like vanilla. I believe the format for saving disks has changed.

**Comment by raoulvdberge** [comment]
The filename changed iirc. Create a clean 1.16.4 world and compare the filenames and location.

**Comment by pwlgrzs** [comment]
@raoulvdberge filenames changed for nodes and networks, but not for the disks file.
@embeddedt so does it mean data is lost?

**Comment by raoulvdberge** [comment]
Do you have any logs?
